pkg/oc/cli/cmd/newapp.go

```diff
@@ -714,24 +714,20 @@ func handleError(err error, baseName, commandName, commandPath string, config *n
 		transformError(err, baseName, commandName, commandPath, groups)
 	}
 	buf := &bytes.Buffer{}
-	if len(config.ArgumentClassificationErrors) > 0 {
-		fmt.Fprintf(buf, "Errors occurred while determining argument types:\n")
-		for _, classErr := range config.ArgumentClassificationErrors {
-			fmt.Fprintf(buf, fmt.Sprintf("\n%s:  %v\n", classErr.Key, classErr.Value))
-		}
-		fmt.Fprint(buf, "\n")
-		// this print serves as a header for the printing of the errorGroups, but
-		// only print it if we precede with classification errors, to help distinguish
-		// between the two
-		fmt.Fprintln(buf, "Errors occurred during resource creation:")
-	}
 	for _, group := range groups {
 		fmt.Fprint(buf, kcmdutil.MultipleErrors("error: ", group.errs))
 		if len(group.suggestion) > 0 {
 			fmt.Fprintln(buf)
 		}
 		fmt.Fprint(buf, group.suggestion)
 	}
+	if len(config.ArgumentClassificationErrors) > 0 && len(groups) > 0 {
+		fmt.Fprintf(buf, "\nFor further assistance,  %s classification of the argument types resulted in the following:\n", commandName)
```

Comment:
I don't think including the cmd name in this output helps (also you've got an extra space after the comma).

I think i'd just trim it down to "Classification of arguments resulted in the following:\n"